frontend/javascripts/oxalis/view/right-border-tabs/bounding_box_tab.js

```diff
@@ -60,8 +130,42 @@ export default function BoundingBoxTab() {
     setPosition(center);
   }
 
+  const showGlobalizeFloodfillsButton =
+    features().jobsEnabled &&
+    activeUser != null &&
+    activeSegmentationTracingLayer != null &&
+    userBoundingBoxes.some(bbox => bbox.name.match(GLOBALIZE_FLOODFILL_REGEX) != null);
+
+  const onGlobalizeFloodfills = newName => {
+    if (activeSegmentationTracingLayer == null) {
+      return;
+    }
+    const baseSegmentationName = getBaseSegmentationName(activeSegmentationTracingLayer);
+    startGlobalizeFloodfillsJob(
+      dataset.owningOrganization,
+      dataset.name,
+      newName,
+      baseSegmentationName,
+      tracing.annotationId,
+      tracing.annotationType,
+    );
+  };
   return (
     <div className="padded-tab-content" style={{ minWidth: 300 }}>
+      {showGlobalizeFloodfillsButton ? (
+        <div style={{ display: "flex", justifyContent: "flex-end" }}>
+          <Tooltip title="For this annotation some floodfill operations were aborted prematurely, because they covered a too large volume. webKnossos can finish these operations via a long-running job.">
```

Comment:
In case you want to use the proposed re-wording

```suggestion
          <Tooltip title="For this annotation some floodfill operations have not run to completion, because they covered a too large volume. webKnossos can finish these operations via a long-running job.">
```